invenio_circulation/signals.py

```diff
@@ -15,12 +15,13 @@
 loan_state_changed = _signals.signal('loan-state-changed')
 """Loan state changed signal.
 
-Its is broadcasted when a loan transitions to a different state.
+Broadcasted when a loan transitions to a different state, sending the old and
```

Comment:
Is it really when a `different state` or rather when a new `action` is applied. I'm mentioning that because the `extend` action does not change the `state` .

Reply:
No, you're right. It's when a new `action` is applied since the state will remain the same for `extend`. Should I update the documentation to make it clearer?

Reply:
If you could, yes. Thanks.